src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -69,9 +82,15 @@ public function supportsNormalization($data, $format = null)
     public function denormalize($data, $class, $format = null, array $context = array())
     {
         $dateTimeFormat = isset($context[self::FORMAT_KEY]) ? $context[self::FORMAT_KEY] : null;
+        $timezone = $this->getTimezone($context);
 
         if (null !== $dateTimeFormat) {
-            $object = \DateTime::class === $class ? \DateTime::createFromFormat($dateTimeFormat, $data) : \DateTimeImmutable::createFromFormat($dateTimeFormat, $data);
+            if (null === $timezone) {
+                // https://bugs.php.net/bug.php?id=68669
```

Comment:
In this case, maybe is it better to check PHP version like in [this example](https://github.com/symfony/symfony/blob/ac5cfee6f12f5c93fe8d8b67941fd478dfaa397a/src/Symfony/Component/Finder/Iterator/RecursiveDirectoryIterator.php#L121-L122)?